src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTabId = tabs.some(({ id }) => id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+
+  const currentTabContent = tabs.find(({ id }) => id === currentTabId).content;
```

Comment:
Use find to pick the object instead and if you don't find any object pick first from the `tabs` array

Reply:
okay!